dynatrace/environment_v2/service_level_objectives.py

```diff
@@ -0,0 +1,282 @@
+"""
+Copyright 2021 Dynatrace LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from enum import Enum
+from requests import Response
+from typing import Dict, Any, Optional
+
+from dynatrace.utils import arg_to_camelcase
+from dynatrace.http_client import HttpClient
+from dynatrace.dynatrace_object import DynatraceObject
+from dynatrace.pagination import PaginatedList
+
+
+class SloService:
+    ENDPOINT = "/api/v2/slo"
+
+    def __init__(self, http_client: HttpClient):
+        self.__http_client = http_client
+
+    def list(
+        self,
+        page_size: Optional[int] = 10,
+        time_from: Optional[str] = "now-2w",
```

Comment:
We have to support `datetime` as well, can you make this an `Optional[Union[datetime, str]]`? 

Then use the `utils.timestamp_to_string()` method when passing it to the params dict